### PR TITLE
true regex filter for paradox

### PR DESCRIPTION
**Pull request by edvorg**

### CODE REVIEW

**Comment by Malabarba** [comment]
Great, I've been meaning to do this as well. Thanks a lot.
Could you replace all mentions of "regex" to "regexp"? That's the usual emacs convention.

File: README.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ itself. They all work out of the box and are completely customizable!
 * Use `paradox-require` instead of `require` to automatically install
   absent packages.
 * Shortcuts for package filtering:
-    * `f r` filters by regexp (`occur`).
+    * `f o` filters by occur.
```
Comment (Malabarba) [inline README.md]: You can remove this line. Now that we have proper regexp filtering, there's no need to document occur.

File: paradox-menu.el [hunk 2]
```diff
@@ -505,6 +508,22 @@ fetching the list.")
 	  (setq paradox--current-filter "Starred")
 	  (paradox-sort-by-package nil))))
 
+(defun paradox-filter-regex ()
+  "Show only packages matching regexp"
+  (interactive)
+  (let* ((regex (read-regexp "Enter regexp: "))
```
Comment (Malabarba) [inline paradox-menu.el]: Could you make this an interactive argument for the function? (Do you know how?)

Comment (edvorg) [inline paradox-menu.el]: No. Could you explain, please?

**Comment by edvorg** [comment]
Fix check it out

File: paradox-menu.el (hunk 2 above)
Comment (edvorg) [inline paradox-menu.el]: Fixed

File: README.md (hunk 1 above)
Comment (edvorg) [inline README.md]: Fixed

**Comment by Malabarba** [comment]
Merged! Thanks again!

**Pull request merged**